BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -26,7 +26,7 @@
     <string name="delete">Löschen</string>
     <string name="rename">Umbenennen</string>
     <string name="change">Ändern</string>
-    <string name="export">Export</string>
+    <string name="export">Export der Bücherliste</string>
```

Comment:
How about "Regal exportieren (BibTeX)"? A bit shorter and I think clearer, as we use the same wording as everywhere else ("Regal" instead of "Bücherliste") and add the export format